extensions/WSLExtension/ClassExtensions/IHostExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace WSLExtension.ClassExtensions;
+
+public static class IHostExtensions

```

Comment:
This, StringExtensions, and ResourceExtensions are copies of what we already have for Hyper-V extension. We could move them to a common place. (Open a bug if not for this PR?).

Reply:
sure will do